### PR TITLE
Clean code: delete commented code

### PR DESCRIPTION
**Pull request by dchittibala**
**What this PR does / why we need it**: #5584

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

File: pkg/lint/rules/template.go [hunk 1]
```diff
@@ -106,11 +106,9 @@ func Templates(linter *support.Linter, values []byte, namespace string, strict b
 			continue
 		}
 
-		// NOTE: disabled for now, Refs https://github.com/kubernetes/helm/issues/1463
 		// Check that all the templates have a matching value
```
Comment (hickeyma) [inline pkg/lint/rules/template.go]: Can you remove the comment and commented out function call?

File: pkg/lint/rules/template.go [hunk 2]
```diff
@@ -106,11 +106,9 @@ func Templates(linter *support.Linter, values []byte, namespace string, strict b
...
 		//linter.RunLinterRule(support.WarningSev, path, validateNoMissingValues(templatesPath, valuesToRender, preExecutedTemplate))
 
-		// NOTE: disabled for now, Refs https://github.com/kubernetes/helm/issues/1037
 		// linter.RunLinterRule(support.WarningSev, path, validateQuotes(string(preExecutedTemplate)))
```
Comment (hickeyma) [inline pkg/lint/rules/template.go]: Can you remove the commented out function call?

**Review by hickeyma — CHANGES_REQUESTED** [review]

**Comment by hickeyma** [comment]
ping @reddydinesh427, any updated on this?

File: pkg/lint/rules/template.go (hunk 1 above)
Comment (reddydinesh427) [reply pkg/lint/rules/template.go]: Which function call needs to be removed?

Comment (bacongobbler) [reply pkg/lint/rules/template.go]: the one on line 110.

**Comment by reddydinesh427** [comment]
@hickeyma updated it

File: pkg/lint/rules/template.go [hunk 3]
```diff
@@ -106,12 +106,7 @@ func Templates(linter *support.Linter, values []byte, namespace string, strict b
 			continue
 		}
 
-		// NOTE: disabled for now, Refs https://github.com/kubernetes/helm/issues/1463
 		// Check that all the templates have a matching value
```
Comment (hickeyma) [inline pkg/lint/rules/template.go]: This line needs to be deleted also.

**Review by hickeyma — CHANGES_REQUESTED** [review]
@reddydinesh427 Thanks for changes. One line left to go. Can you also squash the changes when you do this?

**Review by hickeyma — COMMENTED** [review]
Thanks for cleaning it up @reddydinesh427. Can you do 2 things:
- Sign the DCO: https://helm.sh/blog/helm-dco/index.html
- Squash the commits

**Comment by reddydinesh427** [comment]
@hickeyma updated.

**Review by hickeyma — APPROVED** [review]
@reddydinesh427 LGTM, thanks. I will merge when the circleci build succeeds (looks ok as it seems to be an env issue). I have kicked off another build.

**Pull request merged**